Feature/508 status admin path

subigre opened the pull request:
Update paths for Admin API, Management API and `/status` endpoint

-    Update path for Status endpoint to `/rest/status`.
-    Update base path for Admin API to `/rest/admin`.
-    Update Actuator REST API base path to `/management`.

subigre committed:
Change base path for Spring Boot Actuator endpoints from  status to  management

jakesmolka reviewed (commented):
I can't see changes for the admin API, e.g. here https://ehrbase.readthedocs.io/en/latest/03_development/07_admin/index.html

Same goes for the Status endpoint. But there just wasn't any documentation about those endpoints here, right?

The rest looks good. But I didn't check each (old) "status" endpoint occurrence manually.

subigre committed:
Modifies correct index.rst and updates Postman Collection and Env files

subigre commented:
@jakesmolka 

- Regarding `/status` endpoint, I checked again and did not found any mention in the documentation. 
- I have updated the correct `index.rst` (the one in `/source` 😢).
- Finally, I have updated the Postman Collection and Environment files that I missed the first time

jakesmolka reviewed (approved):
Thanks for the changes! :slightly_smiling_face:
